README.md

```diff
@@ -1,17 +1,145 @@
 <a href="https://opensource.newrelic.com/oss-category/#community-project"><picture><source media="(prefers-color-scheme: dark)" srcset="https://github.com/newrelic/opensource-website/raw/main/src/images/categories/dark/Community_Project.png"><source media="(prefers-color-scheme: light)" srcset="https://github.com/newrelic/opensource-website/raw/main/src/images/categories/Community_Project.png"><img alt="New Relic Open Source community project banner." src="https://github.com/newrelic/opensource-website/raw/main/src/images/categories/Community_Project.png"></picture></a>
 
 # New Relic Kubernetes Agent Operator Init Container Releases
-This repository contains init containers and GHA workflows to automatically release new Kubernetes Agent Operator agent init containers when a new version of an agent becomes available.
-
-## Installation
-
-## Getting Started
+This repository contains init containers and GHA workflows to automatically release new Kubernetes Agent Operator agent init containers when a new version of a language agent becomes available.
 
 ## Usage
 
-## Building
-
-## Testing
+Init containers published here are intended for use with the New Relic [Kubernetes Agents Operator](https://github.com/newrelic/k8s-agents-operator/). See the [Installation](https://github.com/newrelic/k8s-agents-operator/#installation) section for example use cases.
+
+## Image Tagging Conventions
+
+Agent versions at New Relic either follow [semver](https://semver.org/) (eg. `1.2.3`) or may contain an additional build number (eg. `1.2.3.99`). An example table below shows what Python Agent version would be pulled from various tags.
+
+| Image Tag  | Agent Version                                                 |
+|------------|---------------------------------------------------------------|
+| `latest`   | The latest agent release                                      |
+| `9`        | The latest `9.*` release                                      |
+| `9.10`     | The latest `9.10.*` release                                   |
+| `9.10.0`   | The latest **initcontainer build** of agent version `9.10.0`  |
+| `9.10.0.0` | The initial **initcontainer build** of agent version `9.10.0` |
+
+In this example, once the tag `9.10.0.0` is published it will never be replaced. If a rebuilt container for agent verison `v9.10.0` is published, it will be published as `9.10.0.1`, incrementing the initcontainer build number.
```

Comment:
```suggestion
In this example, once the tag `9.10.0.0` is published it will never be replaced. If a rebuilt container for agent version `v9.10.0` is released, it will be published as `9.10.0.1`, incrementing the init container build number.
```